tests/features/sparql_ds.feature

```diff
@@ -0,0 +1,50 @@
+# Date:  10/08/2020
+# Author: Eugeniu Costetchi
+# Email: costezki.eugen@gmail.com
+
+Feature: SPARQL query fetcher
+
+  The data can be fetched from a SPARQL endpoint
+
+  Scenario Outline: Errors of SPARQL select request
+    Given a SPARQL endpoint <endpoint_reference>
+    And a SPARQL query <query_text_reference>
+    When the resultset is requested as tree
+    Then the fetched content should be None
+    And the returned error should contain <error_fragment>
+
+    Examples:
+      | endpoint_reference | query_text_reference              | error_fragment        |
+      | inexistent_server  | spo_limit_10                      | EndPointNotFound      |
+      | crashed_server     | spo_limit_10                      | EndPointInternalError |
```

Comment:
how do you simulate a crashed server?

Reply:
don't yet know :)

Reply:
I don't see it as a test that should be defined as a unit test, rather than an acceptance test